pkg/eventcounter/storage/v2/sql/goal_count.sql

```diff
@@ -0,0 +1,30 @@
+WITH grouped_by_user_evaluation AS (
+    SELECT
+        user_id,
+        variation_id,
+        COUNT(id) as event_count,
+        SUM(value) as value_sum
+    FROM
+        `%s`
+    WHERE
+        _PARTITIONTIME BETWEEN TIMESTAMP(@startAt) AND TIMESTAMP(@endAt)
+    AND environment_namespace = @environmentNamespace
+    AND feature_id = @featureID
+    AND feature_version = @featureVersion
+GROUP BY
+    user_id,
+    variation_id
+)
+SELECT
+    variation_id as variation,
+    COUNT(DISTINCT user_id) as goal_user,
+    SUM(event_count) as goal_total,
+    SUM(value_sum) as goal_value_total,
+    AVG(value_sum) as goal_value_mean,
+    VAR_SAMP(value_sum) as goal_value_variance
```

Comment:
There are two options, `VAR_SAMP` and `VAR_POP` to calculate the variance.
Druid uses `sample` as default rather than `population`, so I chose it.

https://cloud.google.com/bigquery/docs/reference/standard-sql/statistical_aggregate_functions#var_samp

https://druid.apache.org/docs/latest/development/extensions-core/stats.html#pre-aggregating-variance-at-ingestion-time